torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2017,6 +2017,22 @@ def sample_inputs_linalg_qr(op_info, device, dtype, requires_grad=False, **kwarg
         out.append(SampleInput(a))
     return out
 
+def sample_inputs_geqrf(op_info, device, dtype, requires_grad=False):
+    """
+    This function generates input for torch.geqrf
```

Comment:
This comment doesn't hurt anything but I don't think it adds anything to the code, either

Reply:
I agree, we can remove it.